### DISCUSSION TITLE
how to convert a Kolors lora model from .ckpt to .safetensors

### DISCUSSION THREAD

**Discussion by XiangjiBU** [huggingface/diffusers#9617] [Q&A]
how to convert a Kolors lora model from .ckpt to .safetensors

**Comment by a-r-r-o-w** [comment]
I don't think we have conversion scripts for Kolors loras yet. Which trainer are you using, and is this lora available on the HF Hub? cc @asomoza

**Comment by asomoza** [comment]
Probably the kolors lora has only the unet layers, the only difference with SDXL is the text encoder so you can use the same method than any other SDXL lora to convert them if the lora doesn't have text_encoder layers, if it does, then we don't have any method to convert them.
